openrouteservice/src/main/java/org/heigit/ors/routing/RoutingProfile.java

```diff
@@ -887,6 +887,10 @@ else if (bearings[1] == null)
                 req.getHints().put("alternative_route.max_share_factor", searchParams.getAlternativeRoutesShareFactor());
             }
 
+            if(searchParams.hasMaximumSpeed()){
+                req.getHints().put("user_speed", searchParams.getMaximumSpeed());
```

Comment:
Let's be consistent in naming and use the label `maximum_speed` rather than `user_speed` in the hints map as well.